src/overlays/actors/ovl_End_Title/z_end_title.c

```diff
@@ -27,13 +27,134 @@ const ActorInit End_Title_InitVars = {
     (ActorFunc)EndTitle_Update,
     (ActorFunc)EndTitle_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_End_Title/EndTitle_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_End_Title/EndTitle_Destroy.s")
+#include "z_end_title_gfx.c"
+
+void EndTitle_Init(Actor* thisx, GlobalContext* globalCtx) {
+    EndTitle* this = THIS;
+
+    this->theEndTitleCardAlpha = 0;
+    this->zeldaTitleCardAlpha = 0;
+    this->ocarinaTitleCardAlpha = 0;
+    if (this->actor.params == 1) {
+        this->actor.draw = func_80B65DA8;
+    }
+}
+
+void EndTitle_Destroy(Actor* thisx, GlobalContext* globalCtx) {
+}
+
+void EndTitle_Update(Actor* thisx, GlobalContext* globalCtx) {
+}
+
+void EndTitle_Draw(Actor* thisx, GlobalContext* globalCtx) {
+    MtxF* mf;
+    EndTitle* this = THIS;
+    s32 frameCount = globalCtx->csCtx.frames;
+    Player* player = PLAYER;
+    mf = &player->mf_9E0;
+    {
+        GraphicsContext* gfxCtx;
+        Gfx* dispRefs[4];
```

Comment:
the triforce thing is pretty interesting, i think we should comment what the blocks do. "draw a triforce on links left hand" is good for this one and "draw title cards and make the screen sepia" is good for the other imo